src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/DatePicker.xaml

```diff
@@ -135,7 +135,10 @@
                                         Background="Transparent"
                                         BorderBrush="Transparent"
                                         AutomationProperties.Name="{Binding Path=(AutomationProperties.Name), Mode=OneWay, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type DatePicker}}}"
-                                        Cursor="Arrow">
+                                        Cursor="Arrow"
+                                        Focusable="True"
+                                        IsTabStop="True"
+                                        TabIndex="1">
```

Comment:
Do we need tabIndex for keyboard focus? It will interfere the tab order if we explicitly set to 1. @anjali-wpf

Reply:
@sabareesanms - I agree, I have added focus manager and removed tab order.